src/polyglot-notebooks-vscode-insiders/README.md

```diff
@@ -10,18 +10,18 @@ Polyglot Notebooks are fully interoperable with Jupyter and support the `.ipynb`
 
 The following languages are supported by Polyglot Notebooks:
 
-| Language                                    | Variable sharing supported |
-|---------------------------------------------|-------------------------------------------------------|
-| C#                                          |        ✅                 |
-| F#                                          |        ✅                 |   
-| PowerShell                                  |        ✅                 |          
-| JavaScript                                  |        ✅                 |          
-| SQL                                         |        ✅                 |   
-| KQL ([Kusto Query Language](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/))    |        ✅       |       
-| HTML                                        |        ⛔                 |     
-| [Mermaid](https://mermaid.js.org/intro/)    |        ⛔                 |        
-| Python (Preview)                            |        ✅                 |          
-| R (Preview)                                 |        ✅                 |          
+| Language                      | Variable sharing |
```

Comment:
Do we want to add http to this list as well?

Reply:
It's not available yet by default.

Reply:
I was planning a separate PR for that - that should also simplify the setup on the VS-side a bit. I will also update the readmes in that PR.

Reply:
I see, I thought we added it to the default list of kernels - maybe it was a conversation but we haven't really updated the list.

Reply:
The assembly containing the http kernel is included by default as part of the dotnet-interactive tool - this avoids the need for a separate nuget package install for the http functionality (which was required to simplify the VS side setup). However, we haven't yet included the http kernel in the default set of kernels that the dotnet-interactive tool instantiates. As we discussed last week, we should be ready to include this by default now. I'll create a PR for this soon...